### ISSUE TITLE
Rotated tiled 4x 8x8 CJMCU if upgrade firmware

### ISSUE THREAD

**Issue by Giuseppe84** [pixelit-project/PixelIt#195]
Hi everyone. thanks for you Pixelit project. I am using it with four tiled 8x8 CJMCU matrix, and the latest  firmware version  I can properly use is  0.3.20. If I upgrade the firmware and set "type 3 - tiled 4x 8x8 CJMCU" as the matrix type, each matrix appears rotated 90 °, I have tried other "matrix type" settings but none works well. The only solution could be to rotate the matrix, but I can't solve the problem in this way, they are glued on the support.

**Comment by foorschtbar** [comment]
Thats fascinating. My four tiled 8x8 CJMCU matrix doesn't work since i fixed it in 0.4. I think we have connected the tiles differently. Could you send me a picture of the back so I can see how the tiles are connected? I suspect that your pin headers are arranged verticall - this results in a continuous zigzag arrangement across the entire matrix. On my matrix, the pin headers are horizontally, the result is, that the zigzag must jumps after each tile.

tl;dr: send me a picture for confirmation and that we could add a second "8x8 CJMCU matrix" type which work as before.

(DIN is in the top right corner in the image)
![grafik](https://user-images.githubusercontent.com/10727275/188427411-6cc14bd9-3ddc-4f9f-8a37-dec4156b06b5.png)

**Comment by Giuseppe84** [comment]
Of course, I hope the image is clear, unfortunately I put a lot of glue. DIN is in the bottom right corner of each matrix
![0111](https://user-images.githubusercontent.com/5566318/188432220-ff20298f-a753-45f5-b112-a049de6e0be2.jpg)

**Comment by foorschtbar** [comment]
Please test this version. Select Type 3.

[firmware_v1.2.0_d1_mini.zip](https://github.com/pixelit-project/PixelIt/files/9489819/firmware_v1.2.0_d1_mini.zip)

**Comment by Giuseppe84** [comment]
Works perfectly! Thank you very much
